3-extra/1-factorial.js

```diff
@@ -10,10 +10,17 @@
 
 function factorial(input) {
     // TODO
+    if (input === 0 || input === 1) {
+        return 1;
+      } else {
+        for (let i = input - 1; i >= 1; i--) {
```

Comment:
looks good, how could you rewrite this without a `for` loop? (https://www.freecodecamp.org/news/how-to-factorialize-a-number-in-javascript-9263c89a4b38/)